web_responsive/static/src/js/web_responsive.js

```diff
@@ -105,12 +105,12 @@ odoo.define('web_responsive', function(require) {
                 var onIScroll = function() {
                     var transform = (this.iScroll.y) ? this.iScroll.y * -1 : 0;
                     $(this).find('#appDrawerAppPanelHead').css(
-                        'transform', 'matrix(1, 0, 0, 1, 0, ' + transform + ')'
+                        'marginTop', transform + 'px'
                     );
                 };
                 // Scroll probe aggressiveness level
-                // 2 == always executes the scroll event except during momentum and bounce.
-                this.iScroll.options.probeType = 2;
+                // 1 == The scroll event is fired only when the scroller is not busy doing its stuff.
+                this.iScroll.options.probeType = 1;
```

Comment:
This was actually the fix to the header issue - the transform thing I realized marginTop was more clear & basically the same.

Reply:
Hmmm maybe the marginTop isn't the same, I'm reverting that.